legacy/simple_classif_training.py

```diff
@@ -0,0 +1,175 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
What is the point of this file + the other legacy checkpoints?

Reply:
What do you mean?
This is template for creating checkpoint, this one is for master, the copies in version folders are in case some model chnages or needed adjustments to be runnable

Reply:
or would you suggest o use the same source code for all PL versions?